orchagent/portsorch.cpp

```diff
@@ -34,8 +34,10 @@ extern CrmOrch *gCrmOrch;
 
 #define VLAN_PREFIX         "Vlan"
 #define DEFAULT_VLAN_ID     1
-#define FLEX_STAT_COUNTER_POLL_MSECS "1000"
-#define STAT_COUNTER_FLEX_COUNTER_GROUP "STAT_COUNTER"
+#define PORT_FLEX_STAT_COUNTER_POLL_MSECS "1000"
```

Comment:
Are both polling intervals controlled by cli or through fields in config_db.json?
If yes, can you please tell me how?

If not, then those intervals need to have more control.

Reply:
Also queue counters polling should not be enabled by default.

Reply:
@nikos-li , please create feature request in the issue, we'll triage later.

Reply:
@lguohan This isn't a feature request, nor a proper solution to the CPU increase we have observed. This needs to be discussed in the 3-way meeting. Please include it in the agenda.

Reply:
@lguohan @sihuihan88 Please also provide CPU util data of before and after.

Reply:
@lguohan @sihuihan88 Do not merge this without proper RCA and explicit approval and discussion in the 3-way meeting.